content/docs/data_pipeline/introduction/_index.md

```diff
@@ -0,0 +1,77 @@
+---
+weight: 1
+title: "Introduction"
+---
+
+# Introduction
+
+**This is work in progress.**
+
+The FAIR Data Pipeline (FDP) is intended to enable tracking of provenance of data used in epidemiological modelling. File APIs written in C++, Java, Julia, Python and R can be called by model software for data insgestion. These interact with a local relational database storing metadata and the local filesystem, and are configured by a `yml` file associated with the model run. Local files and metadata are synchronised with a remote registry via a command line tool (`fdp`).
+
+The key benefits of using the FAIR Data Pipeline are:
+
+- Provenance of model outputs is tracked to inputs
+- Multiple language support
+- Designed to run on a broad range of platforms (including HPC)
+- Access to shared central data for covid modelling
+
+## Running Models
+
+To use the FAIR Data Pipeline with a piece of modelling software, you must add a language specific API as a dependency and interact with data registered in the pipeline via the methods it presents. Each model run must be configured using a [`config.yml`](../interface/_index.md) file which specifies inputs and outputs by metadata.
+
+{{<mermaid align="left">}}
+graph LR;
+    subgraph Local
+        LR[Local Registry]
+        FS[File System]
+    end
+    subgraph Local API
+        API
+        CY
+    end
+    subgraph Model
+        M
+    end
+
+    LR-->|read_*| API[File API]
+    API-->|write_*| LR
+    FS-->|read_*| API
+    API-->|write_*| FS
+
+    CY[config.yml]-->API
+
+    API-->M[Model]
+    M-->API
+
+{{< /mermaid >}}
+
```

Comment:
This is a cool diagram but it's incomplete. Also, depending on who your target audience is you may wish to add some accompanying text.

Reply:
I believe @richardreeve 's proposed changes have fixed this for now.